### PR TITLE
Added team file

### PR DESCRIPTION
**Pull request by BGull00**

### CODE REVIEW

**Review by ahillhou — APPROVED** [review]

**Review by jAllieB — APPROVED** [review]
Looks good!

**Review by tristan-ainley — APPROVED** [review]

**Comment by argvrutter** [comment]
Duplicate #166

**Comment by argvrutter** [comment]
You already have a PR for your team open: #166 

We created a branch for you to make your team's file on, but if you want to use this PR instead, assign everyone here and close #166

We will only accept a single team PR for a team. All members of the team must be assigned to only the PR you want to use.

**Comment by argvrutter** [comment]
If you are confused on how to modify the content of PR #166 , you would: 
* Clone this repository ( `git clone git@github.com:utk-cs/CS340-Fall-2021.git`) 
* Check out the branch that the PR references (`git checkout team/plumed`)
* Add your team file back in. You could redo your work, via copy/paste, or use git operations for brownie points.

**Comment by jAllieB** [comment]
You mean #166 right?

**Comment by argvrutter** [comment]
> You mean #166 right?

Yep

File: teams/team_plumed.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+* Bryson Gullett
+    * netid: bgullet1
+    * github username: BGull00
+* Tristan Ainley
+    * netid: tainley
+    * github username: tristan-ainley
+* Juliet Bradford
+    * netid: jbradf19
+    * github username: jAllieB
+
```
Comment (argvrutter) [inline teams/team_plumed.md:16]: @jmuncy2 should be listed

**Review by argvrutter — CHANGES_REQUESTED** [review]

File: teams/team_plumed.md (hunk 1 above)
Comment (jAllieB) [reply teams/team_plumed.md:16]: I have added him and (at least attempted to) push teams/team_plumed.md

Comment (jAllieB) [reply teams/team_plumed.md:16]: I pushed it in #166

**Comment by robobenklein** [comment]
#166 merged

**Pull request closed without merging**